### DISCUSSION TITLE
What happens to Vercel dev domain when we add custom domain?

### DISCUSSION THREAD

**Discussion by Prottoy2938** [vercel/vercel#5078] [Help] [answered]
Currently, my application is hosted in `name.now.sh` domain. But soon I will move to a custom domain link like `name.ltd`. So if I move to `name.ltd` domain... what happens to my `name.now.sh` domain? Will the application will be redirected from that domain to the new domain?

My scenario is => suppose I've added `name.ltd` as a custom domain for the application on vercel. So what happens when the user goes to the previous URL: `name.now.sh`? Will they be redirected to the new domain? Or it will show an error?

**Comment by styfle** [comment, marked as answer]
Hi @Prottoy2938 

You can add multiple production domains to a deployment. The same content will be served from both domains, it will _not_ redirect automatically.

If you want to redirect from the old domain to the new domain, you can do that after adding both domains in the dashboard and selecting one to be a redirect.

https://vercel.com/docs/custom-domains#redirecting-domains
